app/controllers/posts_controller.rb

```diff
@@ -9,8 +9,27 @@ def show
     @user = User.find(params[:user_id])
   end
 
+  def create
+    @user = User.find(params[:user_id])
```

Comment:
- [ ] [OPTIONAL] Please modify line 13 and ensure that the person who creates a post is the currently logged in user rather than a user gotten from the `user_id` params. This will enable the reviewer to avoid any confusions when accessing only the actual authorized users are able to delete a post.